src/index.js

```diff
@@ -1 +1,53 @@
-import './style.css';
+import './style.css';
+
+const list = document.getElementById('scorelist');
+
+const myList = [
+  {
+    name: 'Munsa Mibenge',
+    score: 150,
+  },
+  {
+    name: 'Howard Huges',
+    score: 140,
+  },
+  {
+    name: 'Napolean Hill',
+    score: 130,
+  },
+  {
+    name: 'Shannon Biggs',
+    score: 129,
+  },
+  {
+    name: 'Crystal Summer',
+    score: 125,
+  },
+  {
+    name: 'John Lenon',
+    score: 115,
+  },
+  {
+    name: 'Chris Pratt',
+    score: 110,
+  },
+  {
+    name: 'The Rock',
+    score: 108,
+  },
+  {
+    name: 'Mike Myers',
+    score: 103,
+  },
+  {
+    name: 'Eddie Murphy',
+    score: 120,
+  },
+];
+
+myList.forEach((item) => {
+  const li = document.createElement('li');
+
+  li.innerHTML = `${item.name}: ${item.score}`;
+  list.appendChild(li);
+});
```

Comment:
- [ ] Great job on setting up webpack 🥇 Part of the goal for this assignment is however to use ES6 modules to write modular JavaScript with proper ES6 syntax. which includes usage of arrow functions instead of the function keyword. this will make it possible to reuse your code. it will look something like the below example.
```
const index = () => {
  your code....
};

export default index();
```

Reply:
Thanks for the review, Its been handled